datatypes/src/operations/image/colorizer.rs

```diff
@@ -211,6 +229,24 @@ impl Colorizer {
         }
     }
 
+    pub fn over_color(&self) -> Option<RgbaColor> {
+        match self {
+            Colorizer::LinearGradient { over_color, .. }
+            | Colorizer::LogarithmicGradient { over_color, .. }
+            | Colorizer::Palette { over_color, .. } => *over_color,
+            Colorizer::Rgba => None,
+        }
+    }
+
+    pub fn under_color(&self) -> Option<RgbaColor> {
+        match self {
+            Colorizer::LinearGradient { under_color, .. }
+            | Colorizer::LogarithmicGradient { under_color, .. }
+            | Colorizer::Palette { under_color, .. } => *under_color,
+            Colorizer::Rgba => None,
+        }
+    }
```

Comment:
Should be un-optional as well.

Reply:
[1667bc7](https://github.com/geo-engine/geoengine/pull/684/commits/1667bc7a42482d61c00da0265b0de578c593e20f#diff-36b1b82e93e26eceb563391b9c7a3994a3254b9416494a2d64d3b3bfd5da45b1R315) Line: 315